src/test/LocationList.test.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { LocationList } from "@components";
+
+describe("Testing : <LocationList />", () => {
+  let Location1 = [
```

Comment:
should be const mockLocations, and please make the mock data more realistic, eg actual lat/long values.

Reply:
Other than the above changes, very nice work.